brightlayer-ui/react-workflows
# Remove mui/styles dependencies

Fixes BLUI-3661 && #170.


#### Changes proposed in this Pull Request:
- Remove mui/styles dependencies


#### Screenshots:


#### To Test:
- `cd login-workflow && yarn initialize`
- `cd shared-auth && git checkout bug/fix-missing-type-definition-for-react-native`
- `cd .. && yarn start:example`
- ensure the example project still functions as expected using the latest mui/brightlayer-ui dependencies
